src/hyp3_autorift/process.py

```diff
@@ -319,6 +330,40 @@ def apply_landsat_filtering(reference_path: str, secondary_path: str) \
     return reference_path, reference_zero_path, secondary_path, secondary_zero_path
 
 
+def get_lon_lat_from_ncfile(ncfile):
```

Comment:
```suggestion
def get_lat_lon_from_ncfile(ncfile: Path) -> Tuple[float, float]:
```
You should be able to accept a pathlib Path here:
https://unidata.github.io/netcdf4-python/#netCDF4.Dataset

and you're returning lat, lon not lon, lat, so the function name should match